packages/react-liveness/package.json

```diff
@@ -49,11 +49,11 @@
   "dependencies": {
     "@aws-amplify/ui": "6.0.11",
     "@aws-amplify/ui-react": "6.1.5",
-    "@aws-sdk/client-rekognitionstreaming": "3.398.0",
-    "@aws-sdk/util-format-url": "^3.410.0",
-    "@smithy/eventstream-serde-browser": "^2.0.4",
-    "@smithy/fetch-http-handler": "^2.1.3",
-    "@smithy/protocol-http": "^3.0.3",
+    "@aws-sdk/client-rekognitionstreaming": "3.529.1",
```

Comment:
Can we investigate what's happening [here](https://github.com/aws-amplify/amplify-ui/actions/runs/8265948124/job/22613305664?pr=5066#step:14:41)?